### PR TITLE
[264] Querying the Transportation Database

### PR DESCRIPTION
**Pull request by modupeadeonojobi**
Add transportation query file

### CODE REVIEW

File: submissions/modupeAdeOnojobi/database/transportationQuery.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
+1. SELECT survived, COUNT(*) FROM accidents WHERE survived = 1 GROUP BY survived;
+// 290
+
+
+2. SELECT survived, COUNT(*) FROM accidents WHERE survived = 0 GROUP BY survived;
+// 424
```
Comment (adaugochi) [inline submissions/modupeAdeOnojobi/database/transportationQuery.sql:6]: The total number of record is `891` it means when you add `290 + 424`, you ought to have `891` in total. Kindly rerun your query. drop the database, run the query that creates the database and tables again. Then import the passengers CSV file first, then travel detail, before the survival file.

File: submissions/modupeAdeOnojobi/database/transportationQuery.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
+1. SELECT survived, COUNT(*) FROM accidents WHERE survived = 1 GROUP BY survived;
```
Comment (adaugochi) [inline submissions/modupeAdeOnojobi/database/transportationQuery.sql:1]: Just do `SELECT COUNT(*)`. I don't see the reason you are selecting `survived`

File: submissions/modupeAdeOnojobi/database/transportationQuery.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	ON passengers.id = trips.passenger_id WHERE trips.trip_fare > 100 
+	AND trips.number_of_siblings_spouses_aboard > 0 passengers.age > 35;
+
+
+
+5b. // Number passengers...
+	SELECT COUNT(*) FROM passengers LEFT JOIN trips 
+	ON passengers.id = trips.passenger_id WHERE trips.trip_fare > 100 
+	AND trips.number_of_siblings_spouses_aboard > 0 AND passengers.age > 35; 
+// 9
```
Comment (adaugochi) [inline submissions/modupeAdeOnojobi/database/transportationQuery.sql:32]: This is correct

File: submissions/modupeAdeOnojobi/database/transportationQuery.sql [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+4. SELECT COUNT(*), accidents.survived, trips.point_of_embarkation FROM accidents LEFT JOIN trips
+	ON accidents.passenger_id = trips.passenger_id 
+	WHERE trips.point_of_embarkation = 'S' AND accidents.survived = 1;
+// 202
+
+
+5a. // Name and id of passengers...
+	SELECT passengers.id, passengers.full_name FROM passengers LEFT JOIN trips 
+	ON passengers.id = trips.passenger_id WHERE trips.trip_fare > 100 
+	AND trips.number_of_siblings_spouses_aboard > 0 passengers.age > 35;
```
Comment (adaugochi) [inline submissions/modupeAdeOnojobi/database/transportationQuery.sql:24]: There is an error here `trips.number_of_siblings_spouses_aboard > 0 passengers.age > 35`

File: submissions/modupeAdeOnojobi/database/transportationQuery.sql [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+// 424
+
+
+3. SELECT P.full_name, P.sex, P.age, T.point_of_embarkation FROM passengers AS P LEFT JOIN trips AS T 
+	ON P.id = T.passenger_id WHERE P.age < 27 
+	AND (T.point_of_embarkation= 'Q' OR T.point_of_embarkation = 'C');
+
+
+
+4. SELECT COUNT(*), accidents.survived, trips.point_of_embarkation FROM accidents LEFT JOIN trips
```
Comment (adaugochi) [inline submissions/modupeAdeOnojobi/database/transportationQuery.sql:15]: You are only asked to `count` here. This other columns `accidents.survived, trips.point_of_embarkation`, you weren't ask too

File: submissions/modupeAdeOnojobi/database/transportationQuery.sql [hunk 6]
```diff
@@ -0,0 +1,34 @@
+1. SELECT survived, COUNT(*) FROM accidents WHERE survived = 1 GROUP BY survived;
+// 290
+
+
+2. SELECT survived, COUNT(*) FROM accidents WHERE survived = 0 GROUP BY survived;
+// 424
+
+
+3. SELECT P.full_name, P.sex, P.age, T.point_of_embarkation FROM passengers AS P LEFT JOIN trips AS T
```
Comment (adaugochi) [inline submissions/modupeAdeOnojobi/database/transportationQuery.sql:9]: You were only ask to do `P.full_name, P.sex`

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**